wolfcrypt/src/random.c

```diff
@@ -753,9 +753,14 @@ static int _InitRng(WC_RNG* rng, byte* nonce, word32 nonceSz,
         byte seed[MAX_SEED_SZ];
     #endif
 
+#if !defined(WOLFSSL_NO_MALLOC) || defined(WOLFSSL_STATIC_MEMORY)
         rng->drbg =
                 (struct DRBG*)XMALLOC(sizeof(DRBG), rng->heap,
                                                           DYNAMIC_TYPE_RNG);
+#else
+        rng->drbg = (struct DRBG*)rng->drbg_data;
```

Comment:
We need a compile time constant to verify that drbg_data is indeed big enough with future changes.  See CRYPT_MD_Initialize() where the variable md_test performs this function as an example.

Reply:
Fixed pushed. Thanks for catching that. Meant to add that before I did the PR.